api/api/constants/privilege.py

```diff
@@ -0,0 +1,72 @@
+import typing
```

Comment:
What about `AccessLevel`?

Reply:
I like `AccessLevel`!

Reply:
Okay, I've gone with AccessLevel as a clearer name for `Level`, but when I tried it for `Privilege`, it still didn't make semantic sense to me, for similar reasons to `Privilege`. Access level feels like an aspect of what's being described here, rather than the thing itself.

Instead, I came up with `RestrictedFeature`, which I think perfectly captures the thing being described. Let me know what y'all think.

Reply:
Done in https://github.com/WordPress/openverse/pull/4372/commits/8d64c03117994a8bfe721002207a818ba2fac787